tests/semanticdb/metac.expect

```diff
@@ -2886,8 +2886,6 @@ example/NamedApplyBlockMethods.foo().(b) => param b: Int
 example/NamedApplyBlockMethods.foo().(c) => param c: Int
 example/NamedApplyBlockMethods.local. => val method local Int
 example/NamedApplyBlockMethods.recursive(). => method recursive => Int
-local0 => val local c$1: Int
-local1 => val local b$1: Int @uncheckedVariance
```

Comment:
These are lifted arguments from the `recursive` call in `tests/semanticdb/expect/NamedApplyBlock.scala`. Not sure if we want to include them in the symbols list?

Reply:
I would say no. Doesn't seem to be too useful.